### PR TITLE
Connect FAQ Page with Contentful 

### LINKED ISSUE #82: 🙋‍♀️ Integrate Contentful with FAQ Page
**Type**: client
**FOF**:  2

## Description
Integrate Contentful data into the the help page

Utilize this function from Amit's PR (`client/src/api/contentful.ts`) to pull checklist data:
```
const getHelpResources = async (
  state: StateAbbreviation,
): Promise<HelpResource[]> => {
  const res = await getContentfulEntries<HelpResource>('resourceLink', {
    'fields.stateCode': state,
    order: 'fields.order',
  });

  return res.map((r) => r.fields);
};
```

## Todo

- [ ] Pull Contentful Data
- [ ] Integrate data into help page

## Links
- [Contentful API](https://github.com/hack4impact-uiuc/brink/blob/main/client/src/api/contentful.ts)

### PR DESCRIPTION
**Pull request by suewee**
## Status:

:rocket: Ready

## Summary
Integrate Contentful information for the Voter FAQ page.

Closes #82 

## Changes

- Map the FAQ Help Content from Contentful into the Voter FAQ page skeleton
- Make the entire styled accordion clickable as opposed to only the text div
- Resize for mobile responsiveness

### Desktop
image

### Tablet
image

### Mobile
image

### Newline Formatting
image

### CODE REVIEW

**Comment by code-arman** [comment]
Just a thought while looking through this super briefly: is the screen reader able to read the content inside the accordion?

**Comment by amit-sawhney** [comment]
> Just a thought while looking through this super briefly: is the screen reader able to read the content inside the accordion?

I believe it should be able to? I think we tested this with the OG Accordian components

File: client/src/pages/Home.tsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ export const HomepageWrapper: FC<PageWrapper> = ({ page }): ReactElement => {
   }, [pathname]);
 
   return (
-    <main aria-label="home page checklist, faqs, and ballot">
+    // Note: Needed to get rid of nested main tabs, leaving for reference
+    // <main aria-label="home page checklist, faqs, and ballot">
```
Comment (ayanmallik) [inline client/src/pages/Home.tsx:34]: Thanks for removing this outer layer main tag - once we merge other PRs into main we'll want to add back the main tags around the ballot and voting checklist pages.

**Review by ayanmallik — APPROVED** [review]
Awesome job, Sue! It's super cool to see the UI components that we built earlier this semester populated will real data from Contentful.

**Comment by ayanmallik** [comment]
Btw Sue, we're going to want to use a package to render the new lines (`\n`) within Contentful data:
image

Here's a package that Arman referred for solving this issue: https://github.com/nikolovlazar/chakra-ui-prose

**Comment by code-arman** [comment]
> Btw Sue, we're going to want to use a package to render the new lines (`\n`) within Contentful data: image
> 
> Here's a package that Arman referred for solving this issue: https://github.com/nikolovlazar/chakra-ui-prose

I tried settingInnerHTML but it didnt seem to work, so one way to fix this is to do something like this

```
panelText.split("\\n").map((text)=> {
<div> 
{text}
<br/>
</div>

})

```

**Pull request merged**